lib/manticore/client.rb

```diff
@@ -610,23 +612,23 @@ def minimum_encoding_for(string)
 
     # Configure the SSL Context
     def ssl_socket_factory_from_options(ssl_options)
-      trust_store = trust_strategy = nil
+      trust_strategy = nil
 
       case ssl_options.fetch(:verify, :strict)
-      when false
-        trust_store = nil
-        trust_strategy = TrustSelfSignedStrategy::INSTANCE
-        verifier = SSLConnectionSocketFactory::ALLOW_ALL_HOSTNAME_VERIFIER
       when :disable, :none
-        trust_store = nil
         trust_strategy = TrustAllStrategy::INSTANCE
+        verifier = NoopHostnameVerifier::INSTANCE
+      when false # compatibility
+        trust_strategy = TrustSelfSignedStrategy::INSTANCE
         verifier = SSLConnectionSocketFactory::ALLOW_ALL_HOSTNAME_VERIFIER
       when :browser
         verifier = SSLConnectionSocketFactory::BROWSER_COMPATIBLE_HOSTNAME_VERIFIER
-      when true, :strict, :default
+      when true, :default
+        verifier = DefaultHostnameVerifier.new
+      when :strict # compatibility
         verifier = SSLConnectionSocketFactory::STRICT_HOSTNAME_VERIFIER
       else
-        raise "Invalid value for :verify. Valid values are (:all, :browser, :default)"
+        raise "Invalid value for :verify. Valid values are (:default, :browser, :disable)"
```

Comment:
The words `default`, `browser`, and `disable` do not really convey to me the spectrum of their effect, and especially because they are provided as `verify: XXX`, the verb `disable` doesn't really fit with the others which can both be nouns or adverbs depending on context.

I would personally prefer exposing `default`, `browser`, and `none` (and possibly `strict`, even though it is only there for backward-compatibility).